sdk/hybridconnectivity/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
FYI this file is unnecessary unless you add a `tests.yml` file for live testing.